rosdep/base.yaml

```diff
@@ -2850,6 +2850,34 @@ libopenvdb-dev:
   fedora: [openvdb-devel]
   gentoo: [media-gfx/openvdb]
   ubuntu: [libopenvdb-dev, libilmbase-dev]
+liborocos-bfl:
+  debian:
+    '*': [liborocos-bfl0.8]
+    'stretch': null
+  fedora: [liborocos-bfl]
```

Comment:
Is this supposed to be [orocos-bfl](https://fedora.pkgs.org/30/fedora-x86_64/orocos-bfl-0.8.99-17.20180529git3d0d149.fc30.x86_64.rpm.html)?